samza-api/src/main/java/org/apache/samza/system/ExecutionEnvironment.java

```diff
@@ -51,8 +52,10 @@ static ExecutionEnvironment getLocalEnvironment(Config config) {
    */
   static ExecutionEnvironment fromConfig(Config config) {
```

Comment:
QQ: Is this the same ExecutionEnvironment that we were planning to rename it as ApplicationRunner?